reana_client/utils.py

```diff
@@ -410,8 +388,8 @@ def run_command(cmd, display=True, return_output=False, stderr_output=False):
     """
     now = datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S")
     if display:
-        click.secho("[{0}] ".format(now), bold=True, nl=False, fg="green")
-        click.secho("{0}".format(cmd), bold=True)
+        display_message("[{0}] ".format(now), msg_type="success")
+        display_message("{0}".format(cmd))
```

Comment:
*thought*: not sure how important is `bold` in the output. For me, it doesn't make a difference but leaving this comment here just in case.

Reply:
The `display_message` function cannot receive a `bold` flag. We could expose that argument, but I think changing the `display_message` function is not the scope of this PR.

I assumed is not relevant.